src/cityreader/cityreader.py

```diff
@@ -1,6 +1,14 @@
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
+class City:
```

Comment:
"Hard work beats talent when talent fails to work hard."

You did great this week, @taniamichelle ! 
<img width="732" alt="Screen Shot 2019-12-09 at 2 43 26 AM" src="https://user-images.githubusercontent.com/16110931/70416762-c9389300-1a2d-11ea-90d9-72d6903fb722.png">